src/zjs_gpio.c

```diff
@@ -49,6 +50,52 @@ struct gpio_handle {
     uint32_t last;
 };
 
+static jerry_value_t lookup_pin(const jerry_value_t pin_obj, struct device **port, int *pin)
+{
+    char pin_id[12];
```

Comment:
@daniel-thompson : Can you give an example of GPIO device name longer than 8 chars?

@grgustaf : I'll make it 32 as you suggest.